<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Nirmala UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Admin Menu</string>
  </property>
  <property name="windowIcon">
   <iconset theme="logo" resource="logo/logo.qrc">
    <normaloff>:/logo/logo_icon.png</normaloff>:/logo/logo_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{
background-color: qconicalgradient(cx:0.114, cy:0.113318, angle:134.3, stop:0 rgba(214, 28, 30, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QLabel" name="label_Preference_Admin_Menu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>981</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(214, 30, 32);</string>
   </property>
   <property name="text">
    <string>ADMIN MENU</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SEND_EMAIL">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>100</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>SEND E-MAIL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_EVENT_REGISTRATION">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>EVENT REGISTRATION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Return_TO_ADMIN_PREFERENCE_menu">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>500</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>RETURN TO ADMIN PREFERENCE MENU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>500</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(214, 30, 32);
	background-color: rgb(253, 253, 253);
}

QPushButton::hover{
	background-color: rgb(226, 92, 93);
	color: rgb(255, 255, 255);
}

QPushButton::pressed{
	background-color: rgb(216, 37, 39);
}</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>821</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>194</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>28</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>23</number>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Event Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Start Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Participant Email</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Organizer Email</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources>
  <include location="logo/logo.qrc"/>
 </resources>
 <connections/>
</ui>
